test/unittests/commands/excel2json/test_list_node.py

```diff
@@ -62,37 +64,36 @@ def test_to_json(self) -> None:
 
 class TestListRootCreate:
     def test_problem(self) -> None:
-        root = ListRoot.create(id_="", labels={}, comments={}, nodes=[])
-        assert isinstance(root, ListProblem)
-        assert root.root_id == ""
+        root = ListRoot.create(id_="", sheet_name="sheet", labels={}, comments={}, nodes=[])
+        assert isinstance(root, ListSheetProblem)
+        assert root.sheet_name == "sheet"
         assert root.root_problems == {
-            "name": "The name of the list does not contain any characters.",
             "labels": "At least one label per list is required.",
-            "list nodes": "At least one node per list is required.",
```

Comment:
Why has this warning disappeared? I think it would make sense to require at least 1 node.